lib/onebox/templates/githubpullrequest.mustache

```diff
@@ -26,7 +26,7 @@
   <div class="github-info-container">
     {{#commit}}
       <h4>
-        <a href="{{link}}" target="_blank" rel="noopener">{{commit.message}}</a>
+        <a href="{{link}}" target="_blank" rel="noopener">{{commit.message.lines.first}}</a>
```

Comment:
Will the other lines (if present) display in a description sections?

Reply:
No, I did not add it. Do you want me to add one like we have for pull requests?

Reply:
If it's easy to add, it would be great, yes! Thanks!